README.md

```diff
@@ -1,43 +1,85 @@
+![The Buf logo](./.github/buf-logo.svg)
+
 # Buf
 
-[![License](https://img.shields.io/github/license/bufbuild/buf?color=blue)](https://github.com/bufbuild/buf/blob/main/LICENSE)
```

Comment:
I'm not sure about this reordering, but we can talk about it, I'm sure there's reasons

Reply:
I'm not at all invested in this proposed ordering. Just acting on a basic hunch about what people will scan for. Happy to revert.